### PR TITLE
Chapter 12: Password Reset

### PR DESCRIPTION
**Pull request by chienpd-0036**
Chapter 12: Password Reset + I18n + check Rubocop
Screen Shot 2019-07-11 at 01 00 17

### CODE REVIEW

**Comment by chienpd-0036** [comment]
ready

File: app/controllers/password_resets_controller.rb [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  end
+
+  private
+
+  def user_params
+    params.require(:user).permit :password, :password_confirmation
+  end
+
+  def load_user
+    @user = User.find_by email: params[:email]
```
Comment (thanhhungchu95) [inline app/controllers/password_resets_controller.rb]: nếu ko find thấy user thì sao em

File: app/controllers/password_resets_controller.rb [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  end
+
+  def valid_user
+    unless @user&.activated? &&
```
Comment (thanhhungchu95) [inline app/controllers/password_resets_controller.rb]: dùng guard clause

**Comment by chienpd-0036** [comment]
ready

**Pull request merged**